chainstate/src/detail/tests/syncing_tests.rs

```diff
@@ -14,292 +14,268 @@
 // limitations under the License.
 //
 // Author(s): A. Altonen
```

Comment:
I think you  should add S. Tkach here

Reply:
Is that mandatory? It seems kind of pointless to me. Should I add myself to every file where I change one line?.. We already have commits history for that. Perhaps it is needed for some legal reasons?

Reply:
I believe the idea was that if you've made non-trivial changes to the file, you should add your name to the list of authors so people have an idea of who to ask to review the code changes.